.gitignore

```diff
@@ -19,3 +19,6 @@
 *.tgz
 
 .pnpm-debug.log
+
+# auto-generated, see packages/extensions/prepare.mjs
```

Comment:
Maybe clarify that we are the ones auto-generating this - when I started going through this, I thought this was somehow not generated by us.